core/src/avm1/globals/color.rs

```diff
@@ -27,14 +27,12 @@ pub fn constructor<'gc>(
     // The target display object that this color will modify.
     let target = args.get(0).cloned().unwrap_or(Value::Undefined);
     // Set undocumented `target` property
-    this.set("target", target, activation)?;
-    this.set_attributes(
+    this.define_value(
```

Comment:
In principle set+set_attributes aren't always directly replacable by `define_value` (the former also traverses prototype, property setters, watchers etc) - so I'm not 100% comfortable with this change, but in constructors I haven't been able to find an example where it'd break, so I guess it's safe enough...?

Reply:
It's safe in constructors because `this` is a newly-created object, so it doesn't have any watchers/setters that can trigger side effects.